src/test/regress/input/external_table.source

```diff
@@ -131,6 +131,14 @@ FETCH FORWARD 1 FROM _psql_cursor;
 CLOSE _psql_cursor;
 COMMIT;
 
+-- ensure correct truncate error log according to database encoding
+CREATE EXTERNAL WEB TABLE table_test (text text) EXECUTE $$ export LC_ALL=C && echo -n ‘ >/dev/stderr && for i in {1..500}; do echo -n 1 >/dev/stderr; done && echo -n ’ >/dev/stderr && false $$ ON ALL FORMAT 'CSV';
+SELECT true FROM table_test;
+DROP EXTERNAL TABLE IF EXISTS table_test;
+CREATE EXTERNAL WEB TABLE table_test (text text) EXECUTE $$ export LC_ALL=C && echo -n ‘ >/dev/stderr && for i in {1..2006}; do echo -n 1 >/dev/stderr; done && echo -n ’ >/dev/stderr && true $$ ON ALL FORMAT 'CSV';
+SELECT true FROM table_test;
+SELECT count(*) > 0 FROM gp_toolkit.gp_log_system;
```

Comment:
Please recheck this case on unpatched version. There is no log corruption.

Reply:
Also, gp_log_system may scan significant amount of log files. Can we limit this amount? Is there any partitioning?

Reply:
> Please recheck this case on unpatched version. There is no log corruption.
> 
> ```
> test=# CREATE EXTERNAL WEB TABLE table_test (text text) EXECUTE $$ export LC_ALL=C && echo -n ‘ >/dev/stderr && for i in {1..500}; do echo -n 1 >/dev/stderr; done && echo -n ’ >/dev/stderr && false $$ ON ALL FORMAT 'CSV';
> CREATE EXTERNAL TABLE
> test=# SELECT true FROM table_test;
> ERROR:  external table table_test command ended with error. ‘1’  (seg0 slice1 127.0.1.1:6002 pid=11240)
> ```

to get log corruption with false sequence should be up to 2006 not 500!

Reply:
Both scripts write only one symbol 1 to the log.

Reply:
> gp_log_system may scan significant amount of log files. Can we limit this amount?

I add limit

Reply:
> Both scripts write only one symbol 1 to the log.

Hmm... on my setup everything is ok
May I replace generating with results?

Reply:
> Both scripts write only one symbol 1 to the log.

I fix tests